data/yam/agama/auto/default_sle_no_scc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this is not needed, the offline installation seems that is really unclear at the moment, we can hold a bit to understand better how it will be done.

Reply:
I delete it before removing the _WIP_, I'm just using it as a comparison with _SCC_

Reply:
Deleted